File: lib/flammarion/pane.rb
require_relative 'writeable.rb'

module Flammarion
  # A reference to some writeable area within the {Engraving}
  # @see Writeable
  # @see Engraving
  class Pane
    include Writeable

    # @api private
    def initialize(engraving, name, options = {})
      @engraving = engraving
      @pane_name = name
      @options = {}
    end
  end
end

File: nodejs/lib/pane.js
const { Writeable } = require("./writeable");

class Pane extends Writeable
{
    constructor(engraving, name, options = {})
    {
        super()
        this.engraving = engraving
        this.pane_name = name
        this.options = options;
    }
}

module.exports = {Pane}